src/libraries/System.Private.CoreLib/src/Internal/Win32/RegistryKey.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
+using System.Runtime.InteropServices;
 using System.Security;
 
 using Internal.Win32.SafeHandles;
```

Comment:
there's no reasonable way to just use #ifdef to compile a subset of the full implementation, instead of copying part of the code? (I know this was always like this, not suggesting you change it here)